app/controllers/api/v1/merchants/random_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::Merchants::RandomController < ApiBaseController
+  respond_to :json
+
+  def show
+    respond_with Merchant.order("RANDOM()").first
```

Comment:
do we still need to assign the instance variable @merchant? I know it won't be used for any view, but I'm writing my controller tests to take advantage of the assigning of instance variables... we'll see when we go over testing tomorrow